test/test_selector_test.rb

```diff
@@ -7,7 +7,8 @@ def setup
       target_sha = @repo.branches['change_fizz'].target.oid
       master_sha = @repo.branches['master'].target.oid
       base_sha = @repo.merge_base(target_sha, master_sha)
-      @selector = TTNT::TestSelector.new(@repo, target_sha, base_sha)
+      @testtask = TTNT::TestTask.get_instance('test')
```

Comment:
That sounds better to me too... I did it that way since dummy `TTNT::TestTask` object defined that way can have different options from the one defined in the fixture `Rakefile`. But we need to keep the options consistent, and having unneeded method seems worse than managing consistency...

I reverted the commit which added `get_instance` (https://github.com/Genki-S/ttnt/commit/ab05776defffbc2567210f9484a464a8018d6960) and applied your approach to define a dummy `TTNT::TestTask` object (https://github.com/Genki-S/ttnt/commit/8070b7c3c23f1e4d03241f40abfbaa0c9d35c7b6).

(I will squash these commits just before merging)